SpreadCheetah.Test/Helpers/DataValidationGenerator.cs

```diff
@@ -23,6 +23,16 @@ public static IList<DataValidation> Generate(int count)
 
     private static readonly Func<Faker, DataValidation>[] Factories =
     [
+        f => DataValidation.DateTimeBetween(f.Random.SimpleDateTimePair(out var max), max),
+        f => DataValidation.DateTimeEqualTo(f.Random.SimpleDateTime()),
```

Comment:
Maybe we can use the built-in functionality in Bogus to generate DateTimes? For example `f.Date.PastOffset()`.

Reply:
Can you show me an example?

Reply:
`f => DataValidation.DateTimeEqualTo(f.Date.PastOffset())`